core/src/main/java/org/infinispan/configuration/parsing/Attribute.java

```diff
@@ -39,6 +39,7 @@
     CONNECTION_ATTEMPTS,
     CONNECTION_INTERVAL,
     CONSISTENT_HASH_FACTORY,
+    CONTEXT_INITIALIZER,
```

Comment:
We need a bit of context for CONTEXT_INITIALIZER. Let's call it SERIALIZATION_CONTEXT_INITIALIZER